.github/workflows/ci.yml

```diff
@@ -106,7 +106,6 @@ jobs:
         source .venv/bin/activate
         sudo apt-get update
         sudo apt-get install -y libmagic-dev poppler-utils libreoffice
-        make install-pandoc
```

Comment:
removed that redundant call - see few lines below: there's `install-ci` call -> it ensures that `install-pandoc` will run